userops_ext_test.go

```diff
@@ -716,3 +716,39 @@ func Test_Intent_UserOperationString(t *testing.T) {
 		t.Errorf("String() = %v, want %v", result, expected)
 	}
 }
+
+func TestUserOperationRawJSON(t *testing.T) {
+	rawJSON := `{
+        "user_ops": [
+            {
+                "sender": "0x66C0AeE289c4D332302dda4DeD0c0Cdc3784939A",
+                "nonce": "0xf",
+                "initCode": "0x7b2273656e646572223a22307830413731393961393666646630323532453039463736353435633165463262653336393246343662222c226b696e64223a2273776170222c2268617368223a22222c2273656c6c546f6b656e223a22546f6b656e41222c22627579546f6b656e223a22546f6b656e42222c2273656c6c416d6f756e74223a31302c22627579416d6f756e74223a352c227061727469616c6c7946696c6c61626c65223a66616c73652c22737461747573223a225265636569766564222c22637265617465644174223a302c2265787069726174696f6e4174223a307d",
+                "CallData": "{\"sender\":\"0x66C0AeE289c4D332302dda4DeD0c0Cdc3784939A\",\"from_asset\":{\"type\":\"ASSET_KIND_TOKEN\",\"address\":\"0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE\",\"amount\":{\"value\":\"BQ==\"},\"chain_id\":{\"value\":\"AQ==\"}},\"to_stake\":{\"type\":\"ASSET_KIND_STAKE\",\"address\":\"0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84\",\"amount\":{\"value\":\"Mj==\"},\"chain_id\":{\"value\":\"AQ==\"}}}",
```

Comment:
Should we consider the negative case adding a unit test with hex calldata (non-intent userOps) returning `""`?

Reply:
yup...@adelowo